app/containers/ProductContainer.js

```diff
@@ -10,34 +10,42 @@ class ProductContainer extends Component {
         this.state = {
             quantity: 0,
         }
+
         this.handleChange = this.handleChange.bind(this)
         this.handleSubmit = this.handleSubmit.bind(this)
     }
 // handles info typed into quantity box
     handleChange (event) {
-        this.setState({quantity: event.target.value})
+        this.setState({
+            quantity: event.target.value
+        })
     }
 // handles click of Add to Cart button
     handleSubmit (event) {
         event.preventDefault()
         //passing in 'this' was also floated
         this.props.addToCart(this.id, this.state.quantity)
-        this.setState({quantity: 0})
+        this.setState({
+            quantity: 0
+        })
     }
 // passes handlers to component
-    render () {
+    render () {   
```

Comment:
Just a small thing but there is some extra white space after the open curly brace.